apps/genpkey.c

```diff
@@ -332,16 +338,25 @@ static int genpkey_cb(EVP_PKEY_CTX *ctx)
 {
     char c = '*';
     BIO *b = EVP_PKEY_CTX_get_app_data(ctx);
-    int p;
-    p = EVP_PKEY_CTX_get_keygen_info(ctx, 0);
-    if (p == 0)
+
+    if (quiet)
+        return 1;
+
+    switch (EVP_PKEY_CTX_get_keygen_info(ctx, 0)) {
+    default:
```

Comment:
I don't want to reset the clock, but this default case can be removed given the assignment at line 339.

Reply:
It's your changeset...